### PR TITLE
add feat to list the organization scan target groups

### PR DESCRIPTION
**Pull request by Isafagioli-Tenchi**
added method to list all the organization scan targets group

### CODE REVIEW

**Review by edpin — COMMENTED** [review]
Just checking: did you test this? It looks correct, but it's best to validate against dev.

Do you know how to config your cli client?

You need an API key, which you can obtain from the Portal.
Then configure your client with a file in ~/.tenchi/config. Then create a new section with the profile name:

```
[testing-dev] # dev, not PR environment
api_url = https://api.zanshin.tenchi-dev.com
api_key = <api_key>
```

Then in the cli, use `zanshin --profile testing-dev ...`

**Review by asieira — DISMISSED** [review]

**Review by edpin — DISMISSED** [review]

File: zanshincli/main.py [hunk 1]
```diff
@@ -1006,6 +1006,14 @@ def organization_scan_target_scan_get(
 organization_app.add_typer(scan_target_group_app, name="scan-target-groups",
                    help="Operations on organizations scan target groups the API key owner has direct access to")
 
+@scan_target_group_app.command(name='list')
+def scan_target_groups_list(organization_id: UUID = typer.Argument(..., help="UUID of the organization")):
+    """
+    Lists the scan target groups of the organization user.
```
Comment (edpin) [inline zanshincli/main.py:1012]: I think you mean: "Lists the scan target groups of the user's organization"

**Review by edpin — APPROVED** [review]

File: zanshincli/main.py [hunk 2]
```diff
@@ -1015,6 +1014,18 @@ def scan_target_groups_list(organization_id: UUID = typer.Argument(..., help="UU
...
     output_iterable(client.iter_organization_scan_target_groups(organization_id))
 
+@scan_target_group_app.command(name='create')
+def scan_target_groups_create(
+        organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+        kind: ScanTargetKind = typer.Argument(..., help="kind of the scan target group. Should be 'ORACLE'" ),
+        name: str = typer.Argument(..., help="name of the scan target group")
+):
+    """
+    Create a scan target group of the organization.
```
Comment (edpin) [inline zanshincli/main.py:1024]: s/of/for/

**Review by edpin — APPROVED** [review]

File: zanshincli/main.py (hunk 2 above)
Comment (Isafagioli-Tenchi) [reply zanshincli/main.py:1024]: sorry, I didn't understand

Comment (edpin) [reply zanshincli/main.py]: Ah, using `sed` syntax: replace "of" for "for":

"Creates a scan target group for the organization"

Comment (Isafagioli-Tenchi) [reply zanshincli/main.py]: oh ok, it's done now

**Pull request merged**